content/department_filials/filial_agricultural.md

```diff
@@ -0,0 +1,29 @@
+title: Филиал кафедры экономики и управления предприятиями АПК
+slug: filial_agricultural
+category: department_filials
+tags: [_nav, _category_header]
+---
+
+__Организация, в которой функционирует филиал:__
+
+РНУП «Институт системных исследований в АПК НАН Беларуси»
+
+__Год создания филиала:__
+
+2014
+
+__Основные направления деятельности филиала:__
+
+1.	привлечение сотрудников Института к участию в учебной и учебно-методической работе;
```

Comment:
This should be unnumbered list, since each list item starts with a lowercase letter and ends with a semicolon `;`. Make sure there is __only one__ space between `-` and a sentence.